Formula/gismo.rb

```diff
@@ -0,0 +1,66 @@
+class Gismo < Formula
+  desc "C++ library for isogeometric analysis (IGA)"
+  homepage "https://gismo.github.io"
+  url "https://github.com/gismo/gismo/archive/refs/tags/v21.12.0.tar.gz"
+  sha256 "4001b4c49661ca8b71baf915e773341e115d154077bef218433a3c1d72ee4f0c"
+  license "MPL-2.0"
+
+  head do
+    url "https://github.com/gismo/gismo.git", branch: "stable"
+  end
+
+  keg_only "we don't want all examples in install path"
```

Comment:
Can't we exclude the examples?

Reply:
Unfortunately, all examples and the few utility executables like `gsView` are in the examples folder. In the current CMake configuration we can only decide to build them all or none of them. So, an alternative is to disable building examples and remove the `keg_only` flag. Being one of the G+Smo developers we can separate examples from utilities in a forthcoming release (more quickly in HEAD).

Reply:
So let's disable all of them. I think being keg-only is worse than not having examples